example/redis-sentinel.yaml

```diff
@@ -8,6 +8,9 @@ spec:
   podSecurityContext:
     runAsUser: 1000
     fsGroup: 1000
+  pdb:
+    enabled: false
+    minAvailable: 2
```

Comment:
can you make the `minAvailable: 2` to be 1

Reply:
okay